shared/middleware/helpers/storage_apps.rb

```diff
@@ -13,10 +13,12 @@ def initialize(storage_id)
   end
 
   def create(value, ip_address)
+    timestamp = Time.now
```

Comment:
before we start populating this field with Time.now, I would like to understand whether we are standardizing on either Time.now or DateTime.now, or whether it is safe to have a mix of them in different places. Is this something you feel you have wrapped your head around?